src/components/restaurant/RestaurantCard.tsx

```diff
@@ -1,7 +1,7 @@
+import { Place } from '@googlemaps/google-maps-services-js';
```

Comment:
I'm wondering, instead of this library are we able to make use of https://www.npmjs.com/package/@types/googlemaps since it's already installed? I think there's a `PlaceResult` interface here that looks similar to the one you're using in this file

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/googlemaps/reference/places-service.d.ts

Reply:
ah yes, we could use it, but BE is currently sending data in the format of the `Place` interface. `PlaceResult` seems like the same, but I'm not sure if I wanna take the risk that they'll remain the same in the future.

The reason why I chose this library in the BE is cause of the following pros where old=@types/googlemaps and new=@googlemaps/google-maps-service-js
![image](https://user-images.githubusercontent.com/49849754/110419833-eb64fa80-804e-11eb-8197-f55d228d59c6.png)

Reply:
Ohhh didn't know this, that's good to know! Yeah in that case it's probably better to stick with @googlemaps/google-maps-service-js